models/gc_tiger_lb2/dockerfiles/Dockerfile

```diff
@@ -20,9 +20,11 @@ RUN apt-get update \
 # Install tiger LB2 algorithm
 #   - Clone tiger LB2 codebase (master branch, fixed to commit 720f8dfca4624792c8e57915c4222efec5a0c2d4)
 #   - Subsequently we remove the .git directory to procuce a compacter docker layer
+#   - We also fix a reproducibility issue with the segmentation prediction by fixing the torch random seed
 RUN git clone https://github.com/vuno/tiger_challenge.git /vuno && \
     cd /vuno && git reset --hard 720f8dfca4624792c8e57915c4222efec5a0c2d4 && \
-    rm -rf /vuno/.git
+    rm -rf /vuno/.git && \
+    sed -i "238i \        torch.manual_seed(0)" /vuno/segmentation/inference.py
```

Comment:
This is great but should be implemented in the source repository.

Reply:
Yes, ideally we would be able to make the necessary changes in the original repository. However, since it is unlikely that we have that option, we need to discuss the best approach to take. One possible solution is forking the repository and implementing the changes in the forked version. Let's discuss this in more detail during our next meeting. For now, let's focus on the remaining changes and come up with a solid plan to address this issue as soon as possible.

Reply:
I have forked the repository and added the change there. The manual correction is no longer in the Dockerfile.